hw_3_zd_2.js

```diff
@@ -0,0 +1,21 @@
+const sarah = {
+    name: 'Sarah',
+    job: {
+        name: 'Andersen',
+        hiringDate: '23.07.2020'
+    }
+}
+
+const {
```

Comment:
don't use global variables. Make destructurization with input params directly inside the function

Reply:
Done

Reply:
Done

Reply:
Done